jcommune-view/jcommune-web-view/pom.xml

```diff
@@ -142,7 +142,7 @@
       <plugin>
         <groupId>ro.isdc.wro4j</groupId>
         <artifactId>wro4j-maven-plugin</artifactId>
-        <version>1.7.5</version>
+        <version>1.7.6</version>
```

Comment:
In the commit message could you please add an explanation on why updating the version helps.